src/App.tsx

```diff
@@ -1,27 +1,53 @@
 import React from 'react';
 import './App.scss';
+import { Counter } from './Counter';
 
-const App: React.FC = () => {
-  const timerId: NodeJS.Timer = setInterval(() => {
-    const date: Date = new Date();
+class App extends React.Component {
+  state = {
+    // eslint-disable-next-line react/no-unused-state
+    time: new Date().toLocaleTimeString(),
+    // eslint-disable-next-line react/no-unused-state
+    timerId: undefined,
```

Comment:
Delete this properties from the state of App component
They should be in the state of Clock component